src/isomorphic/React.js

```diff
@@ -109,8 +110,11 @@ if (__DEV__) {
       get() {
         lowPriorityWarning(
           didWarnPropTypesDeprecated,
-          'Accessing PropTypes via the main React package is deprecated. Use ' +
-            'the prop-types package from npm instead.',
+          'In React v16.0+, accessing PropTypes via the main React package is deprecated. Use ' +
```

Comment:
We could rephrase "In React 16.0+, accessing PropTypes from React package will no longer work. [...]" Then in my opinion it's clearer what will happen and when.